GitUI/BranchTreePanel/RepoObjectsTree.ContextActions.cs

```diff
@@ -122,6 +122,15 @@ private void ContextMenuBranchSpecific(ContextMenuStrip contextMenu)
             _localBranchMenuItems.GetInactiveBranchItems().ForEach(t => t.Item.Visible = isNotActiveBranch);
 
             _menuBranchCopyContextMenuItems.ForEach(x => x.Visible = node.Visible);
+
+            if (node.Visible)
+            {
+                contextMenu.AddUserScripts(runScriptToolStripMenuItem, scriptKey => _runScriptCommand.Execute(scriptKey));
```

Comment:
```suggestion
                contextMenu.AddUserScripts(runScriptToolStripMenuItem, _runScriptCommand.Execute);
```
would reduce noise. Though I have not checked whether it works in C#.